apps/mull-ui/src/app/pages/profile/edit-profile/edit-profile.tsx

```diff
@@ -41,11 +41,19 @@ const EditProfile = ({ history }: EditProfilePageProps) => {
         .test('EmojiCheck', 'Emojis are not allowed in Display Name.', function (displayName) {
           return !hasEmoji(displayName);
         }),
-      description: Yup.string().max(
-        LIMITS.DESCRIPTION,
-        `Description must be ${LIMITS.DESCRIPTION} characters or less.`
-      ),
       imageFile: Yup.mixed().test('big-file', 'File size is too large', validateFileSize),
+      description: Yup.string()
+        .max(
+          LIMITS.PROFILE_DESCRIPTION,
+          `Description must be ${LIMITS.PROFILE_DESCRIPTION} characters or less.`
+        )
+        .test(
+          'NbOfLinesCheck',
+          `Description must not exceed ${LIMITS.LINES} lines.`,
+          function (description) {
+            return exceedsNbOfLines(description);
+          }
+        ),
```

Comment:
es6 syntax to make it shorter and easier to read.
```suggestion
        .test(
          'NbOfLinesCheck',
          `Description must not exceed ${LIMITS.LINES} lines.`, (description) => exceedsNbOfLines(description)
        ),
```